frontend/config-overrides.js

```diff
@@ -0,0 +1,6 @@
+const rewireStyledComponents = require("react-app-rewire-styled-components");
+
+module.exports = function override(config, env) {
+  config = rewireStyledComponents(config, env);
```

Comment:
This is so our React components have human-readable names in the React dev tools extension.